shared/domains/chelonia/internals.js

```diff
@@ -801,7 +845,9 @@ export default (sbp('sbp/selectors/register', {
                     key: deserializeKey(decrypted),
                     transient
                   }]))
-                  if (
+                  if (missingDecryptionKeyIds?.includes(key.id)) {
```

Comment:
Can you add a comment here explaining what this is doing?

Reply:
```
                  // If we've just received a known missing key (i.e., a key
                  // that previously resulted in a decryption error), we know
                  // our state is outdated and we need to re-sync the contract
```